nca/Utils/CmdlineRunner.py

```diff
@@ -9,23 +9,32 @@
 
 import subprocess
 import os
+import sys
 from fnmatch import fnmatch
 
 
 class CmdlineRunner:
     """
     A stateless class with only static functions to easily get k8s and calico resources using kubectl and calicoctl
     """
+    # a static variable to indicate if we want to ignore errors from running executable command
+    ignore_live_cluster_err = False
+
     @staticmethod
     def run_and_get_output(cmdline_list):
         """
         Run an executable with specific arguments and return its output to stdout
+        if a communicate error occurs, it will be ignored in case this is a silent try to communicate with live cluster,
+        otherwise, will be printed to stderr
         :param list[str] cmdline_list: A list of arguments, the first of which is the executable path
-        :return: The executable's output to stdout
+        :return: The executable's output to stdout ( a list-resources on success, otherwise empty value)
         :rtype: str
         """
-        cmdline_process = subprocess.Popen(cmdline_list, stdout=subprocess.PIPE)
-        return cmdline_process.communicate()[0]
+        cmdline_process = subprocess.Popen(cmdline_list, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        out, err = cmdline_process.communicate()
+        if err and not CmdlineRunner.ignore_live_cluster_err:
```

Comment:
This function (`run_and_get_output`) gets called also by `resolve_helm_chart`, and on this case the flag `ignore_live_cluster_err` is not relevant.